src/app/executive/layout.module.css

```
@@ -0,0 +1,158 @@
+}
+
+.navLinkTitle {
+  min-width: 0;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.externalBadge {
+  flex: 0 0 auto;
+  border: 1px solid currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)